doc/user/interop.md

```diff
@@ -85,8 +95,33 @@ Otherwise, return the instance variable names, without the leading `@`.
 
 `KEYS(other)` → `other.instance_variables.map { |key| ...key without @... }`
 
-In both cases the keys are returned as a Ruby `Array` containing Ruby `String`
-objects.
+In both cases if the `internal` flag is set, instance variable names with the
+leading `@` are also returned.
+
+Keys are returned as a Ruby `Array` containing Ruby `String` objects.
+
+### `KEY_INFO`
+
+If the object is a Ruby `Hash`, and the key is in the hash, `READABLE`,
+`WRITABLE` are set. If the key is not in the hash, neither are set. Note that
```

Comment:
"`READABLE`, `WRITABLE` are set" -> "`READABLE` and `WRITABLE` are set"?

Reply:
Should `WRITABLE` be set if the hash is frozen?

Reply:
Fixed.